packages/react/src/patterns/sub-patterns/PictogramItem/README.md

```diff
@@ -54,16 +59,27 @@ ReactDOM.render(<App />, document.querySelector('#app'));
 | ----------- | -------- | --------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `heading`   | YES      | String    | null          | Heading text                                                                                                                                            |
 | `copy`      | YES      | String    | null          | Copy text                                                                                                                                               |
-| `Pictogram` | YES      | Component | null          | Pictogram component imported from `@carbon/pictograms-react` package                                                                                    |
+| `pictogram` | YES      | Object    | null          | See `Pictogram` bellow. package                                                                                                                         |
 | `cta`       | NO       | Object    | null          | Object with CTA data, check [CTA](https://github.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/react/src/components/CTA/README.md)👀 |
 
+## pictogram
+
+| Name | Required | Data Type | Default Value | Description | | `src` | YES |
+Component | null | Pictogram component imported from `@carbon/pictograms-react`
+
+## Accessibility
+
+In order to make the component acessible to screen readers, aria-label and
+aria-hidden are available for you to use, adding `ariaLabel` or `ariaHidden`
+values as a string inside the `pictogram`object.
```

Comment:
I'm not sure `ariaHidden` needs to be called out.  Just say something like "provide an aria attribute". Then give an example, and then you can link to Carbon's documentation on the matter.

https://github.com/carbon-design-system/carbon/tree/master/packages/icons-react#focus-and-aria-label